src/util.h

```diff
@@ -593,6 +622,19 @@ enum ESvrRegResult
     SRR_CENTRAL_SVR_FULL = 1
 };
 
+inline QString svrRegResultToString ( ESvrRegResult eSvrRegResult )
```

Comment:
Why do we have this additional function? Isn't svrRegStatusToString() enough?

Reply:
There are two enums.  It's mostly to ensure the behaviour is correct - it wasn't entirely clear from the report that raised this.  This logs when the Central Server response is received; status gets logged when the Slave server changes state - either from receiving a response, or not.